aws/resource_aws_launch_template.go

```diff
@@ -358,21 +362,19 @@ func resourceAwsLaunchTemplate() *schema.Resource {
 			},
 
 			"security_group_names": {
-				Type:     schema.TypeSet,
```

Comment:
Similar note here about `security_group_names` and `vpc_security_group_ids` -- we shall see!